### ISSUE TITLE
Revolver Variants broken on launching the game Vanila

### ISSUE THREAD

**Issue by Liamconn35** [wafflethings/WafflesWeapons#30]
I believe it's because some of the default revolver variants on my save were left on variant priority #5 which doesn't exist in vanilla, but sharpshooter and piercer cannot be used and can also not have their priority numbers changed in any way. I would just boot up the mod and bring them down into the default three priority numbers but as we all know the mod is currently broken by the Violence update, anyone know how I might fix this without having to wait for the mod being updated or am I just stuck with marksman and nothing else until then?

**Comment by ShyGuy120** [comment]
Have you ever found a fix for this?

**Comment by Liamconn35** [comment]
Nope, I've deleted every bit of the game files including my saves, and then reinstalled them from steam before, but somehow the glitch remained. I honestly don't have the vaguest idea how this could be fixed.

**Comment by Liamconn35** [comment]
> Have you ever found a fix for this?

I actually did, here are the steps.
1. Make a copy of any saves you want to hold on to
2. Delete everything inside the Programfiles(x86)/Steam/Steamapps/Common/Ultrakill folder
3. put this into the steam Console without the quotation marks "download_depot 1229490 1229491 1141582293364162456"
4. Wait for the file download to complete fully
5. take everything from the Programfiles(x86)/Steam/Steamapps/content/app_1229490/depot_1229491 folder and move them into the Programfiles(x86)/Steam/Steamapps/Common/Ultrakill folder.
6. Run the game from your steam library page without any pre-existing saves to make sure variants that were unusable are now fixed. This may take a while if the broken variants are further into the game (I lucked out since the piercer was one of my broken variants)
7. If the broken variants are fixed without your old saves you can transfer any saves you have back into the Programfiles(x86)/Steam/Steamapps/Common/Ultrakill/saves folder and everything should be fully back to normal.

I can't say that this method is foolproof but it worked for me and it's better than nothing.
